A32NX/html_ui/JS/A32NX_Util.js

```diff
@@ -46,6 +46,59 @@ A32NX_Util.createMachine = (machineDef) => {
     return machine;
 };
 
+/**
+ * Takes a LatLongAlt or LatLong and returns a vector of spherical co-ordinates
+ * @param {LatLong | LatLongAlt} ll
+ */
+A32NX_Util.latLonToSpherical = (ll) => {
+    return [
+        Math.cos(ll.lat * Avionics.Utils.DEG2RAD) * Math.cos(ll.long * Avionics.Utils.DEG2RAD),
+        Math.cos(ll.lat * Avionics.Utils.DEG2RAD) * Math.sin(ll.long * Avionics.Utils.DEG2RAD),
+        Math.sin(ll.lat * Avionics.Utils.DEG2RAD)
+    ]
+}
+
+/**
+ * Computes the intersection point of two bearings on a great circle
+ * @param {LatLong | LatLongAlt} latlon1
```

Comment:
Is this correct way for a union of multiple types?

Reply:
Yes: https://jsdoc.app/tags-param.html#multiple-types-and-repeatable-parameters